x-pack/plugins/security_solution/common/api/endpoint/metadata/list_metadata.gen.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { z } from 'zod';
+
+/*
+ * NOTICE: Do not edit this file manually.
+ * This file is automatically generated by the OpenAPI Generator `yarn openapi:generate`.
+ */
+
+import { Page, PageSize } from '../model/schema/common.gen';
+
+export type ListRequestQuery = z.infer<typeof ListRequestQuery>;
+export const ListRequestQuery = z.object({
+  page: Page.and(z.unknown()).and(z.unknown()),
+  pageSize: PageSize.and(z.unknown()).and(z.unknown()).and(z.unknown()),
+  kuery: z.string().nullable().optional(),
+  sortField: z
+    .enum([
+      'enrolled_at',
+      'metadata.host.hostname',
+      'host_status',
+      'metadata.Endpoint.policy.applied.name',
+      'metadata.Endpoint.policy.applied.status',
+      'metadata.host.os.name',
+      'metadata.host.ip',
+      'metadata.agent.version',
+      'last_checkin',
+    ])
+    .optional(),
+  sortDirection: z.enum(['asc', 'desc']).nullable().optional(),
+  hostStatuses: z.array(z.unknown()).optional(),
```

Comment:
why is this `z.unknown` ? Should it be `type:array` or `enum` and not both?

Reply:
Same question for `page` and `pageSize` above

Reply:
Yeah I think these got wrongly generated. I'll take another look at it. I think it didn't import type properly. 
Thanks for pointing me in this direction 👍

Reply:
Fixed 👍